src/brasil/gov/barra/tests/test_helper.py

```diff
@@ -1,41 +1,27 @@
 # -*- coding: utf-8 -*-
-try:
-    # Python 3
-    from urllib.request import Request, urlopen
-except ImportError:
-    # Python 2
-    from urllib2 import Request, urlopen
-
 from brasil.gov.barra.browser.barra import BarraViewletJs
-from brasil.gov.barra.config import BARRA_JS_DEFAULT_LANGUAGE
-from brasil.gov.barra.config import BARRA_JS_FILE
 from brasil.gov.barra.config import BARRA_JS_STATIC_FILE_LOCATION
-from brasil.gov.barra.config import BARRA_JS_URL
 from brasil.gov.barra.testing import INTEGRATION_TESTING
-from filecmp import cmp
 from plone import api
-from time import time
-from zope.component import getMultiAdapter
 
+import os
 import unittest
 
 
-BARRA_EXTERNA_HTML = u'<script defer="defer" src="//barra.brasil.gov.br/barra.js" type="text/javascript"></script>'
-BARRA_LOCAL_HTML = u'<script defer="defer" src="++resource++brasil.gov.barra/barra.js" type="text/javascript"></script>'
+BARRA_JS_URL = 'https://barra.brasil.gov.br/barra_2.0.js'
+BARRA_JS_FILE = 'barra.js'
+BARRA_JS_STATIC_FILE_LOCATION = os.path.join(
+    BARRA_JS_STATIC_FILE_LOCATION, BARRA_JS_FILE)
+BARRA_JS_DEFAULT_LANGUAGE = 'pt-BR'
+
+BARRA_EXTERNA_HTML = u'<script defer src="//barra.brasil.gov.br/barra_2.0.js"></script>'
+BARRA_LOCAL_HTML = u'<script defer src="++resource++brasil.gov.barra/barra.js"></script>'
 
 
 class HelperViewTest(unittest.TestCase):
     """ Caso de teste da Browser View BarraHelper"""
-    layer = INTEGRATION_TESTING
 
-    def language_lowercase(self):
```

Comment:
@caduvieira @idgserpro @claytonc não entendi a função desse método aqui e estou removendo por considerar desnecessário.

Reply:
São constantes usadas para forçar a chamada de linguagem específica da barra como se fosse um navegador, pois a barra é multilingue. Ver https://github.com/plonegovbr/brasil.gov.barra/issues/26#issuecomment-256045090

Serve inclusive para a chamada curl que era feita pra atualizar a barra. Não sei se o que você fez em https://github.com/plonegovbr/brasil.gov.barra/commit/ffa8c639a4c6f815382e0846b65800b20decd4d3#diff-a55fc4e553af13234f26342646b88e19 contempla isso.

Particularmente, estou achando que esse PR está tratando de duas coisas diferentes.

Reply:
Veja o blame nesse arquivo e verá que esse método foi adicionado para resolver https://github.com/plonegovbr/brasil.gov.barra/issues/42.

Reply:
bom se foi para fazer o teste passar então não precisa mais pois o teste está passando.